### PR TITLE
Fix error - Inject empty meta tag when addon not enabled

### PR DESCRIPTION
**Pull request by kamilio**
Since the update from `0.7.0`  to `0.9.0`, I've been getting the following error in environments where the `ember-cli-ifa` is **not** enabled. 

Leadfeeder - Loading  2020-01-29 15-18-29

After digging through the source, I discovered that the `meta` tag is[ added regardless](https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/index.js#L33) of whether the addon is enabled or not. Then, the function `getAssetMapData` tries to [parse the placeholder](https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/addon/utils/get-asset-map-data.js#L17) `__ember-cli-ifa__AssetMapPlaceholder__` that never gets replaced as the addon is not enabled. 

I've modified the part where the meta tag gets injected. It injects empty tag without the placeholder which causes an early return in function that parses JSON but doesn't trigger the missing meta warning.

### CODE REVIEW

**Comment by Turbo87** [comment]
I'm not sure I understand the issue, but it sounds like https://github.com/adopted-ember-addons/ember-cli-ifa/pull/100 should have already fixed this 🤔

**Comment by kushthedude** [comment]
Hey @Turbo87 , This PR actually solves our issue too. We are trying to update cli-ifa from 0.6.0 to 0.9.0, we are using ember-cli-ifa in our production environment and when we try to build the app in development we experience similar type of error @kamilio mentioned(`SyntaxError: JSON Parse error: Unexpected identifier "__ember"). Not to mention this PR solves our issue of same. Will it be possible to get it merged any time soon ?

**Comment by kamilio** [comment]
@Turbo87 thanks for the comment. PR #100 solves a different problem when the `meta` tag is not added at all. I assume that it's for cases when `content_for` is missing in the template. 

However, our case is different. The meta tag is added but the Placeholder, magic string, that was supposed to be replaced by valid JSON is not being replaced.

**Comment by kamilio** [comment]
Hello @RuslanZavacky @Turbo87 

How are you doing? Would you have some time to look over this PR?

**Comment by kamilio** [comment]
@mansona hello :) would you have a minute?

**Comment by alexlafroscia** [comment]
@kamilio I can get this merged for you (and am running into the same problem myself now)

Can you just update the PR to remove the `package-lock.json` file? This project is already configured to use `yarn` and we don't want to have both lockfiles present.

**Comment by Turbo87** [comment]
is it possible for us to test against regressions for this?

**Comment by alexlafroscia** [comment]
Hmm... Maybe configuring the dummy app's `environment.js` to accept an override of the `enabled` property, and adding an `ember-try` configuration with the add-on disabled?

**Comment by kamilio** [comment]
@alexlafroscia thanks for coming back to me

I removed the `package-lock.json` and force pushed changes, however, the CI didn't run, like it did when I first opened the PR. I even pushed another commit, just to make sure.

**Comment by alexlafroscia** [comment]
Tests are failing, but that doesn’t seem related to these changes at all. It appears some dependency requires Node 10 or higher, but we’re using Node 8 to test against. That’s something I can open a PR for to resolve...

I would be comfortable merging this without explicit tests, but I would love an OK from @Turbo87 first to make sure that sounds reasonable. I’m not sure how realistic it would be to add an `ember-try` scenario in the way that I suggested, not because it would be hard, but because we would probably want to run a different set of tests for the addon in that case... The need for multiple dummy apps strikes again!

File: index.js [hunk 1]
```diff
@@ -55,10 +59,7 @@ module.exports = {
   postBuild(build) {
     this._super.included.apply(this, arguments);
 
-    const env = process.env.EMBER_ENV;
-    const ifaConfig = this.project.config(env).ifa;
```
Comment (Turbo87) [inline index.js]: why did you remove the `ifaConfig` variable here? removing it means we run `this.project.config()` multiple times, which might be expensive in a few cases. maybe we should at least cache the result?

**Comment by kamilio** [comment]
Closing this as I enabled `ember-cli-ifa` in all environments.
